source/config/profileUpgradeSteps.py

```diff
@@ -84,3 +84,16 @@ def upgradeConfigFrom_4_to_5(profile):
 	except KeyError:
 		# Setting does not exist, no need for upgrade of this setting
 		log.debug("reportDetails not present, no action taken.")
+
+
+def upgradeConfigFrom_5_to_6(profile):
+	"""
+	useInMSWordWhenAvailable in UIA section has been replaced with allowInMSWord multichoice.
+	"""
+	try:
+		useInMSWord = profile['UIA']['useInMSWordWhenAvailable']
+		del profile['UIA']['useInMSWordWhenAvailable']
+	except KeyError:
+		useInMSWord = False
+	if useInMSWord:
+		profile['UIA']['allowInMSWord'] = 3  # ALWAYS

```

Comment:
Could we use the enum here?
```suggestion
		profile['UIA']['allowInMSWord'] = AllowUiaInMSWord.ALWAYS
```

Reply:
While I agree using an enum here makes this much more readable the suggestion cannot be applied as is:
1. We have to use `.value` on an enum member when adding its value to config.
2. More importantly importing `UIAHandler` in `config` (that is much earier in NVDA's startup process than it is done now) would almost certainly prevent NVDA from starting. This suggest to me that the enum in question should be placed in `config` rather than in `UIAHandler` and `UIAHandler` can just reuse it from there.

Reply:
I'll move both AllowInMSWord and AllowInChromium enums to configspec.py and rename them to reflect they are for UIA.